packages/access-api/test/helpers/context.js

```diff
@@ -70,7 +73,7 @@ export async function context(options) {
   return {
     mf,
     conn: connection({
-      principal,
+      principal: servicePrincipal,
```

Comment:
Nit: Can you please delete `principal` created on line 53 then ?

Reply:
It's used in the return value [on line 82](https://github.com/web3-storage/w3protocol/pull/334/files#diff-dd020d0285601ca47fabbd295beaa017b1415699c44343d1c183ef9f849341c8R82).

Before I touched this, it would use the same generated signer both for the connection servicePrincipal as well as the returned `issuer`. afaict that would mean most tests invocations before had identical `iss` and `aud`, which isn't necessarily wrong, but seems rare.

now it uses both, but it **does** use both so I believe I should not delete the var called `principal`, unless I'm missing something and you think I should return a different value for `context.issuer`

Reply:
Thanks for clarification I did miss it on line 82 and yeah I would expect `iss === aud` to be an unrealistic scenario so having them differ is good 👍